scripts/publish-root.gradle

```diff
@@ -0,0 +1,38 @@
+// Used to sign the artifacts after they are built
+ext["signing.keyId"] = ''
+ext["signing.password"] = ''
+ext["signing.key"] = ''
+// Account details for Maven Central
+ext["ossrhUsername"] = ''
+ext["ossrhPassword"] = ''
+
+ext["sonatypeStagingProfileId"] = ''
+
+File secretPropsFile = project.rootProject.file('local.properties')
```

Comment:
can we add this file to the .gitignore if it's not already there?

Reply:
It's already there 👍